developer/testing/velocity.md

```diff
@@ -21,7 +21,7 @@ As such, our testing falls into two locations:
    You enable these tests by uncommenting the velocity testing packages in `.meteor/packages`.
```

Comment:
This line is no longer true as these packages are not in the packages files any longer.